src/App.vue

```diff
@@ -1,16 +1,34 @@
 <template>
   <div id="app">
     <h1>{{ msg }}</h1>
+    <app-new-item @itemAdded="newItem"></app-new-item>
```

Comment:
Super jedeš pojmenvání dle styleguidu.
Taky raději používam 'App' než 'Base'
Používá se to ale tak, aby název komponenty byl alespoň dvouslovný. Když už název obsahuje 2 slova tak se ten prefix nepřidává.
Dělá se to pro to, abys nevytvářel názvy komponent který by mohli konfliktit s html